test/passing/eliom_ext.eliom

```diff
@@ -0,0 +1,34 @@
+let%server log str = Lwt_io.write_line Lwt_io.stdout str
+
+let%client log = ~%(Eliom_client.server_function [%derive.json: string] log)
+
+let%client () =
+  Eliom_client.onload
+    (* NB The service underlying the server_function isn't available on the
+       client before loading the page. *)
+    (fun () ->
+      Lwt.async (fun () -> log "Hello from the client to the server!") )
+
+[%%shared
+type some_type = int * string list [@@deriving json]
+
+type another_type = A of some_type | B of another_type [@@deriving json]]
```

Comment:
The trailing `]` is not very visible. Would one introduce a break before ?

Reply:
I think we can think about making it affected by `indicate-multiline-delimiters` but not in this PR indeeed